### PR TITLE
Chore/rss ecomm 1 12 initialize husky

### LINKED ISSUE #16: Issue RSS-ECOMM-1_12: Initialize Husky 🐶
Description 📝
Initialize Husky to manage Git hooks, automating tasks such as code formatting and linting checks during the commit process. 📝➡️🔎

Acceptance Criteria 📚
Husky is initialized and integrated into the project. ✔️
Git hooks are managed by Husky, automating code formatting and linting checks. ✔️🔄

### PR DESCRIPTION
**Pull request by Tetiana-KET**
#### issue RSS-ECOMM-1_12: Initialize Husky 🐶
resolve #16

#### 🤔 This is a 

- [x] 🌟 New task - Husky and hooks setup
- [ ] ⚙️ Update to an existing task
- [ ] 🛠 Fix in a task or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] ❓ Other 

#### Description
Initialize Husky to manage Git hooks, automating tasks such as code formatting and linting checks during the commit process. 📝➡️🔎

- **Acceptance Criteria:** 📚
- [x] Husky is initialized and integrated into the project. ✔️
- [x] Git hooks are managed by Husky, automating code formatting and linting checks. ✔️🔄

#### Additional Information

- **Screenshots/Links:**
 image
image

#### Checklist
**The only open issue, I did all the checks for the precommit. Shall i move them to pre-push?**

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.
- [ ] 👌 Have at least 1 approve.
- [x] 🌳 Branch from `develop`/`release/...` and target to `develop`/`release/...`.

### CODE REVIEW

**Review by Maksim99745 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "preview": "vite preview",
     "ci:format": "npx prettier src/ --check",
     "format": "npx prettier src/ --write",
```
Comment (Gromov-Roman) [inline package.json]: please rename this to "prettier" and above to "prettier:fix"

File: package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
     "preview": "vite preview",
     "ci:format": "npx prettier src/ --check",
```
Comment (Gromov-Roman) [inline package.json]: just `prettier src/ --check` doesn't work?

File: .lintstagedrc.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "./src/.": ["npm run ci:format"],
+  "./src/**.*": ["npm run format"],
```
Comment (Gromov-Roman) [inline .lintstagedrc.json]: I believe we can't do fix automatically, please remove

File: package.json (hunk 1 above)
Comment (Gromov-Roman) [reply package.json]: Suggestion:
```json
"prettier": "prettier --check --ignore-unknown .",
"prettier:fix": "prettier --write --ignore-unknown .",
```

Comment (Gromov-Roman) [reply package.json]: or `src/` instead of `.`

**Review by Gromov-Roman — APPROVED** [review]

**Pull request merged**